depending on install-all here results in install being called without
setting PREFIX to the fake-dir. we already call install-all from port
building anyway.

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -60,8 +60,8 @@ check-style:
 .PHONY: reindent check-style
 
 # depend on install-all so that downgrade scripts are installed as well
-check: all install-all
+check: all
 	# explicetely does not use $(MAKE) to avoid parallelism
-	make -C src/test/regress check
+	gmake -C src/test/regress check
 
 .PHONY: all check clean install install-downgrades install-all
